bin/cron/ops_data_pull

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+
+require_relative '../../dashboard/config/environment'
+require 'cdo/properties'
+include ApplicationHelper
+include UsersHelper
+
+pd_scripts = Script.all.select { |script| !!script.pd }
+pd_scripts.each do |script|
+  # Get all users with any activity in the script
+  users = User.joins('left join user_scripts on user_scripts.user_id = users.id').where(user_scripts: {script_id: script})
```

Comment:
hm, I notice that we don't have an index in that direction:

````
irb(main):020:0> User.joins('left join user_scripts on user_scripts.user_id = users.id').where(user_scripts: {script_id: script}).explain
=> EXPLAIN for: SELECT `users`.* FROM `users` left join user_scripts on user_scripts.user_id = users.id WHERE `users`.`deleted_at` IS NULL AND `user_scripts`.`script_id` = 77
+----+-------------+--------------+--------+---------------------------------------------+---------------------------------------------+---------+-------------------------------------------+---------+--------------------------+
| id | select_type | table        | type   | possible_keys                               | key                                         | key_len | ref                                       | rows    | Extra                    |
+----+-------------+--------------+--------+---------------------------------------------+---------------------------------------------+---------+-------------------------------------------+---------+--------------------------+
|  1 | SIMPLE      | user_scripts | index  | index_user_scripts_on_user_id_and_script_id | index_user_scripts_on_user_id_and_script_id | 8       | NULL                                      | 7861725 | Using where; Using index |
|  1 | SIMPLE      | users        | eq_ref | PRIMARY                                     | PRIMARY                                     | 4       | dashboard_production.user_scripts.user_id |       1 | Using where              |
+----+-------------+--------------+--------+---------------------------------------------+---------------------------------------------+---------+-------------------------------------------+---------+--------------------------+
2 rows in set (0.00 sec)

irb(main):021:0> UserScript.where(script_id: script).explain
=> EXPLAIN for: SELECT `user_scripts`.* FROM `user_scripts`  WHERE `user_scripts`.`script_id` = 77
+----+-------------+--------------+------+---------------+------+---------+------+---------+-------------+
| id | select_type | table        | type | possible_keys | key  | key_len | ref  | rows    | Extra       |
+----+-------------+--------------+------+---------------+------+---------+------+---------+-------------+
|  1 | SIMPLE      | user_scripts | ALL  | NULL          | NULL | NULL    | NULL | 7861725 | Using where |
+----+-------------+--------------+------+---------------+------+---------+------+---------+-------------+
1 row in set (0.00 sec)

````

Adding that index could speed up this query.